### PR TITLE
NumberFormatException When replacing Image in ODT file using Velocity

### PR DESCRIPTION
**Pull request by arons**

### CODE REVIEW

File: document/fr.opensagres.xdocreport.document/src/test/java/fr/opensagres/xdocreport/document/images/FileImageProviderTestCase.java [hunk 1]
```diff
@@ -146,36 +146,36 @@ public void loadPNGByAndChangeResize()
...
         // Force height and compute width.
         imageProvider.setSize( null, 1000f );
-        Assert.assertNotNull( imageProvider.getWidth(0) );
-        Assert.assertEquals( 1100f, imageProvider.getWidth(0).floatValue(), 0 );
-        Assert.assertNotNull( imageProvider.getHeight(0) );
-        Assert.assertEquals( 1000f, imageProvider.getHeight(0).floatValue(), 0 );
+        Assert.assertNotNull( imageProvider.getWidth(null) );
+        Assert.assertEquals( 1100f, imageProvider.getWidth(null).floatValue(), 0 );
+        Assert.assertNotNull( imageProvider.getHeight(null) );
+        Assert.assertEquals( 1000f, imageProvider.getHeight(null).floatValue(), 0 );
```
Comment (angelozerr) [inline document/fr.opensagres.xdocreport.document/src/test/java/fr/opensagres/xdocreport/document/images/FileImageProviderTestCase.java]: Could you add a test with 0.582cm like https://github.com/opensagres/xdocreport/issues/187

Comment (arons) [inline document/fr.opensagres.xdocreport.document/src/test/java/fr/opensagres/xdocreport/document/images/FileImageProviderTestCase.java]: Maybe in a new test class? the test is more about AbstractImageRegistry.getSize(String sizeAsDxa) method. Let me know, if you prefer having it in FileImageProviderTestCase I can add also there.

Comment (angelozerr) [inline document/fr.opensagres.xdocreport.document/src/test/java/fr/opensagres/xdocreport/document/images/FileImageProviderTestCase.java]: > Maybe in a new test class? the test is more about AbstractImageRegistry.getSize(String sizeAsDxa) method. 

Yes you right, please create a new test class. Even if we doesn't support cm for the moment, we will have a test which doesn't creash the report even with cm.

**Comment by angelozerr** [comment]
Many thanks @arons !

**Pull request merged**